package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "task1",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon ./app.js --exec babel-node --presets babel-preset-env,stage-2",
```

Comment:
extract babel configs to `.babelrc` file than have just short `nodemon --exec babel-node ./app.js`